cfgov/ask_cfpb/models/pages.py

```diff
@@ -241,6 +244,210 @@ def page_js(self):
         return js
 
 
+class PortalSearchPage(
+        RoutablePageMixin, SecondaryNavigationJSMixin, CFGOVPage):
+    """
+    A routable page type for Ask CFPB portal search ("see-all") pages.
+    """
+
+    objects = CFGOVPageManager()
+    portal_topic = models.ForeignKey(
```

Comment:
Could you specify `related_name='portal_search_pages'` or something equivalent, to make the reverse relation a little nicer? Without specifying something, the default on PortalTopic objects would be something like `PortalTopic.portalsearchpage_set`, which isn't as night.

It also occurs to me that, if there's always a 1:1 relationship between PortalSearchPages and PortalTopics, in theory this could be a [OneToOneField](https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.OneToOneField), but that might cause problems with, say, experimenting with other pages or having old pages to the trash.

Reply:
- 1:1
    - The portals are in flux, to put it mildly, and some pages are likely to be trashed as it plays out.

- related_name
    - √